redish/libs/redish-frontend/authentication/data-access/src/lib/connector/authentication-api.service.ts

```diff
@@ -1,27 +1,49 @@
-import { CreateUserDTO } from '@redish-shared/domain';
+import {
+  AuthenticateUserDto,
+  CreateUserDto,
+  RedishError,
+  TokenDto,
+  UuidDto,
+} from '@redish-shared/domain';
 import axios from 'axios';
 
-export class AuthenticationService {
-  private baseURL = 'https://localhost:3000/auth';
+export class AuthenticationApiService {
+  // todo: move to config
+  private baseURL = 'http://localhost:3000/user';
 
   public async register(
     username: string,
     email: string,
     password: string
   ): Promise<string | null> {
-    const url = `${this.baseURL}/login`;
-    const data: CreateUserDTO = { username, password, email };
+    const url = `${this.baseURL}/register`;
```

Comment:
Url is not correct anymore, has moved to just a post on '/user'

```suggestion
    const url = `${this.baseURL}`;
```